### PR TITLE
doc: add Electron 30 update section

### PR DESCRIPTION
**Pull request by deepak1556**

### CODE REVIEW

File: release-notes/v1_92.md [hunk 1]
```diff
@@ -498,6 +498,14 @@ In this milestone, we picked up the work again to adopt ESM for VS Code core. Ou
...
+The Node version with this Electron update contains a [breaking change](https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2#command-injection-via-args-parameter-of-child_processspawn-without-shell-option-enabled-on-windows-cve-2024-27980---high), in response to a CVE, which may affect you if you execute `.bat` or `.cmd` files on Windows. You can follow the [guidelines](https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows) set by Node.js when spawning these files using the `shell` option. We have tried to avoid any surprises for our extension community by proactively notifying of this change by a simple scan of the extension source code https://github.com/aws/aws-toolkit-vscode/issues/5292, https://github.com/GoogleCloudPlatform/cloud-code-vscode/issues/920, https://github.com/dotnet/vscode-dotnet-runtime/issues/1870. It is possible we might have not caught all the impacted extensions if the extension source does not deal with these files but might have an indirect dependency at runtime. We recommend extension authors to always test using our [insiders version](https://code.visualstudio.com/insiders/) to catch these changes in advance.
```
Comment (connor4312) [inline release-notes/v1_92.md:507]: I would replace

> We have tried ... dependency at runtime

with

> We have proactively notified extensions who may be affected based on a simple source code scan, but it's possible with may have missed yours. If you are affected, you will encounter an `EINVAL` exception when spawning `.bat` or `.cmd` files on Windows. Please refer to the [Node.js documentation](https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows) for guidance on how to handle these.

I worry that linking to specific issues on other repos will lead to confused people looking for help or bothering other maintainers

Comment (ntrogh) [reply release-notes/v1_92.md:507]: +1 on Connor's remarks.

 There's a small typo:
> it's possible with may have missed yours -> it's possible **we** may have missed yours

Comment (ntrogh) [inline release-notes/v1_92.md:507]:
```suggestion
The Node version with this Electron update contains a [**breaking change**](https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2#command-injection-via-args-parameter-of-child_processspawn-without-shell-option-enabled-on-windows-cve-2024-27980---high), in response to a CVE, which may affect you if you execute `.bat` or `.cmd` files on Windows. You can follow the [guidelines](https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows) set by Node.js when spawning these files using the `shell` option. 

We have proactively notified extensions who may be affected based on a simple source code scan, but it's possible we may have missed yours. If you are affected, you will encounter an EINVAL exception when spawning `.bat` or `.cmd` files on Windows. Please refer to the [Node.js documentation](https://nodejs.org/api/child_process.html#spawning-bat-and-cmd-files-on-windows) for guidance on how to handle these.

We recommend extension authors to always test their extension with our [Insiders release](https://code.visualstudio.com/insiders/) to catch these changes in advance.
```

**Review by ntrogh — COMMENTED** [review]
Adding suggestion from @connor4312, as @deepak1556 is OOF today.

**Comment by ntrogh** [comment]
@connor4312 Can you review my edits and confirm if they can be merged?

File: release-notes/v1_92.md (hunk 1 above)
Comment (1chooo) [reply release-notes/v1_92.md]: 👍🏻 Wow, that’s very thoughtful and considerate!

**Comment by deepak1556** [comment]
Thanks @connor4312 @ntrogh

**Pull request merged**